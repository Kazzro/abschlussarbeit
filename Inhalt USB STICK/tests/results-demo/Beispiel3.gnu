reset
set term png size 900,1394
set output 'Beispiel3.png'
set xrange [0:900]
set yrange [0:1294]
set size ratio -1

set title "\
// Auftrag IHK3\n\
Benötigte Länge: 129.4cm\n\
Genutzte Fläche: 89.90%"

set style fill transparent solid 0.5 border
set key noautotitle

$data <<EOD
# x_LU y_LU x_RO y_RO Auftragsbeschreibung ID
0 0 100 100 "Auftrag A" 1
0 100 880 400 "Auftrag D" 4
0 400 210 697 "Auftrag B" 2
210 400 630 697 "Auftrag C" 3
630 400 840 697 "Auftrag E" 5
100 0 200 100 "Auftrag F" 6
0 697 210 994 "Auftrag G" 7
210 697 630 994 "Auftrag H" 8
0 994 880 1294 "Auftrag I" 9
630 697 840 994 "Auftrag J" 10
EOD

$anchor <<EOD
# x y
880 100
840 400
0 697
210 697
630 697
200 0
100 100
210 994
880 994
0 1294
840 697
630 994
EOD

plot \
'$data' using (($3-$1)/2+$1):(($4-$2)/2+$2):(($3-$1)/2):(($4-$2)/2):6 \
              with boxxy linecolor var, \
'$data' using (($3-$1)/2+$1):(($4-$2)/2+$2):5 \
              with labels font "arial,9", \
'$anchor' using 1:2 with circles lc rgb "red", \
'$data' using 1:2 with points lw 8 lc rgb "dark-green"
