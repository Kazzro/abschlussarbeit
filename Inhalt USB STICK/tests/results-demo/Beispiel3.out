// Auftrag IHK3
Benötigte Länge: 129.4 cm
Benutzte Flaeche: 89.90 %

Positionierung der Kundenaufträge:
    0 0 100 100 - 1 - Auftrag A
    0 100 880 300 - 4 - Auftrag D
    0 400 210 297 - 2 - Auftrag B
    210 400 420 297 - 3 - Auftrag C
    630 400 210 297 - 5 - Auftrag E
    100 0 100 100 - 6 - Auftrag F
    0 697 210 297 - 7 - Auftrag G
    210 697 420 297 - 8 - Auftrag H
    0 994 880 300 - 9 - Auftrag I
    630 697 210 297 - 10 - Auftrag J

Verbleibende Andockpunkte:
    880 100
    840 400
    0 697
    210 697
    630 697
    200 0
    100 100
    210 994
    880 994
    0 1294
    840 697
    630 994
